rnd/autogpt_server/autogpt_server/agent_api/__init__.py

```diff
@@ -0,0 +1 @@
+from .server import start_server  # noqa
```

Comment:
What's the `# noqa` for?

Reply:
Make the linter happy, it warns that `start_server` is not used.